### PR TITLE
New combinatorial.cpp

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

**Comment by SATYAPAL-BISHNOI** [comment]
,

File: test/doc/CMakeLists.txt [hunk 1]
```diff
@@ -25,8 +25,8 @@ glob_unit("doc" ${doc_root} "bessel/regular/*.cpp")
 ##==================================================================================================
 add_custom_target(doc.combinatorial.exe        )
 add_dependencies(doc.exe doc.combinatorial.exe )
-glob_unit("doc" ${doc_root} "combinatorial/regular/*.cpp"    )
-glob_unit("doc" ${doc_root} "combinatorial/conversion/*.cpp" )
+glob_unit("doc" ${doc_root} "combinatorial/*.cpp"    )
+##glob_unit("doc" ${doc_root} "combinatorial/conversion/*.cpp" )
```
Comment (jfalcou) [inline test/doc/CMakeLists.txt:29]: remove donc comment

File: test/unit/module/combinatorial/fibonacci.cpp [hunk 2]
```diff
@@ -24,15 +24,3 @@ TTS_CASE_TPL("Check return types of eve::fibonacci", eve::test::simd::ieee_reals
   TTS_EXPR_IS(eve::fibonacci(i_t(), elt_t(), T()), T);
   TTS_EXPR_IS(eve::fibonacci(std::uint8_t(), elt_t(), elt_t()), elt_t);
 };
-
-//==================================================================================================
-// Test for corner-cases values
-//==================================================================================================
-TTS_CASE_TPL("Check corner-cases behavior of eve::fibonacci on wide", eve::test::simd::ieee_reals)
```
Comment (jfalcou) [inline test/unit/module/combinatorial/fibonacci.cpp:31]: put it back maybe ?

**Review by jfalcou — CHANGES_REQUESTED** [review]

File: test/doc/CMakeLists.txt (hunk 1 above)
Comment (jtlap) [reply test/doc/CMakeLists.txt:29]: done

File: test/unit/module/combinatorial/fibonacci.cpp (hunk 2 above)
Comment (jtlap) [reply test/unit/module/combinatorial/fibonacci.cpp:31]: done

**Review by jfalcou — APPROVED** [review]
